molgenis-data/src/main/java/org/molgenis/data/transaction/TransactionConfig.java

```diff
@@ -16,10 +17,14 @@
 	@Autowired
 	private MolgenisTransactionManager transactionManager;
 
+	// This cannot be a app setting because apps settings are not loaded yet when we use it
+	@Value("${use.transaction.log:true}")
```

Comment:
If it takes a lot of time then we should improve the performance of logging transactions (which doesn't sound like a heavy job). This will improve the all over performance of the web application. This solution feels like a workaround for the actual problem.